pkg/katautils/network.go

```diff
@@ -179,3 +180,25 @@ func hostNetworkingRequested(configNetNs string) (bool, error) {
 
 	return false, nil
 }
+
+// cleanupNetNS cleanup netns created by kata, trigger only create sandbox fails
+func cleanupNetNS(netNSPath string) error {
+	n, err := ns.GetNS(netNSPath)
+	if err != nil {
+		return err
+	}
+
+	err = n.Close()
+	if err != nil {
+		return err
```

Comment:
Can you return a more verbose message here : "Failed to close netns: %s"

Reply:
Updated.